movies/movies

```diff
@@ -158,28 +166,16 @@ getConfiguredPython || exit 1
 getConfiguredClient || exit 1
 checkInternet || exit 1 # check if we have a valid internet connection if this isnt true the rest of the script will not work so stop here
 
-while getopts "uvh" opt; do
-  case $opt in
-    \?)
-      echo "Invalid option: -$OPTARG" >&2
-      exit 1
-    ;;
-    h)
-      usage
-      exit 0
-    ;;
-    v)
-      echo "Version $currentVersion"
-      exit 0
-    ;;
-    u)
-      update
-      exit 0
-    ;;
-    :)
-      echo "Option -$OPTARG requires an argument." >&2
-      exit 1
-    ;;
+while getopts 'ud:hv' flag; do
```

Comment:
So the way I set up the flag was for usage like: _movies -d Rogue One_. The colon changes what happens if you simply ran _movies -d_. Without the colon it will search for the movie "-d", with the colon it exits the program saying _option requires an argument -- d_. Not really a big deal either way, whatever you prefer.